sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/SmallITCluster/cassandra-svc-rc.yaml

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
if the difference between the small and large cluster is just that there's a different number of replicas, we should think about consolidating them down to one yaml with a different shell script to invoke it.